License2Deploy/rolling_deploy.py

```diff
@@ -209,8 +216,8 @@ def setup_logging(): # pragma: no cover
 
 def main(): # pragma: no cover
   setup_logging()
-  check_args()
-  l = RollingDeploy(argv[1], argv[2], argv[3], argv[4], argv[5])
+  a = check_args()
+  l = RollingDeploy(a.env, a.project, a.buildNum, a.amiID, a.profile, a.config)
```

Comment:
What does 'a' stand for? Apples?
I think it would be better to name the variable as opposed to using a single letter.
principle from 'clean code'
``` python 
args = get_args()
```
Not a showstopper though